File: evals/code-intelligence/fixtures/batch-d/swift/Sources/App/ItemService.swift
final class ItemService: ItemLoading {
    func find(id: String) -> Item {
        Item(id: id)
    }
}

extension ItemService {
    func summary(item: Item) -> String {
        item.id
    }
}

func describe(service: ItemService, item: Item) -> String {
    service.summary(item: item)
}

File: evals/code-intelligence/fixtures/batch-e/php/src/ItemService.php
<?php
namespace App;

use App\Contracts\ItemLoader as LoaderContract;
use App\Support\LogsItems;

final class ItemService extends BaseService implements LoaderContract
{
    use LogsItems;

    public function find(string $id): Item
    {
        return new Item($id);
    }

    public function summary(Item $item): string
    {
        return $item->id;
    }
}

function describe(ItemService $service, Item $item): string
{
    return $service->summary($item);
}
